python/libcuvs/pyproject.toml

```diff
@@ -106,4 +106,4 @@ select = [
 ]
 
 # detect when package size grows significantly
-max_allowed_size_compressed = '1.0G'
+max_allowed_size_compressed = '1.3G'
```

Comment:
This the spot for general configuration of the size limit, but because of `nccl` vendoring differences between cuda 11 and cuda 12, the limits are changed (depending on cuda version) in `ci/validate_wheel/sh` -- and please do not bump the limit by any more than is strictly necessary

Reply:
Got it, thank you